src/main/java/com/commercetools/sync/services/impl/CustomObjectServiceImpl.java

```diff
@@ -53,29 +53,31 @@ public CustomObjectServiceImpl(@Nonnull final CustomObjectSyncOptions syncOption
         return fetchCachedResourceId(identifier.toString(),
             draft -> CustomObjectCompositeIdentifier.of(draft).toString(),
             () -> CustomObjectQuery.ofJsonNode()
-                        .withPredicates(q -> q.container().is(container).and(q.key().is(key)))
+                                   .withPredicates(q -> q.container().is(container).and(q.key().is(key)))
         );
     }
 
     @Nonnull
     @Override
     public CompletionStage<Set<CustomObject<JsonNode>>> fetchMatchingCustomObjects(
-            @Nonnull final Set<CustomObjectCompositeIdentifier> identifiers) {
+        @Nonnull final Set<CustomObjectCompositeIdentifier> identifiers) {
 
         Set<CustomObjectCompositeIdentifier> filteredIdentifiers = identifiers.stream()
-                .filter(identifier ->
-                            StringUtils.isNotEmpty(identifier.getContainer())
-                        &&  StringUtils.isNotEmpty(identifier.getKey()))
-                .collect(Collectors.toSet());
+                                                                              .filter(identifier ->
```

Comment:
the previous formatting on this section satisfied my eyes more 👀

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/569/commits/29234abafea0a7d15eae352184ee69787e59fc19